Glaze/manifest.json

```diff
@@ -1,11 +0,0 @@
-{
-    "name": "Glaze",
-    "description": "Glaze",
-    "preview": "screenshots/dark.png",
-    "readme": "README.md",
-    "usercss": "user.css",
-    "schemes": "color.ini",
-    "authors": [
-        { "name": "CharlieS1103", "url": "https://github.com/CharlieS1103" }
-    ]
-}
```

Comment:
Shouldn't we add to the root `manifest.json`?

Reply:
Nope, hosted in my own repo

Reply:
Why is it on this one then?